doc/source/ray-core/tasks/scheduling.rst

```diff
@@ -48,9 +65,29 @@ until the resource utilization is beyond a certain threshold and spread tasks af
         # Spread tasks across the cluster.
         [spread_function.remote() for i in range(100)]
 
+        @ray.remote
+        def node_function():
+            return 3
+
+        # Only run the task on the local node.
+        node_function.options(
+            scheduling_strategy=NodeSchedulingStrategy(
+                node_id = ray.get_runtime_context().node_id.hex(),
+                soft = False,
+            )
+        ).remote()
+
+        # Run the task on the remote node if possible.
+        node_function.options(
+            scheduling_strategy=NodeSchedulingStrategy(
```

Comment:
on naming: will `NodeAffinityStrategy` be a better name?

Reply:
Yea, sounds a better name! Should it be `NodeAffinitySchedulingStrategy` or it's too long.

Reply:
yeah `NodeAffinityStrategy` seems having the right balance.

Reply:
@stephanie-wang @ericl wdyt?